content/docs/2.3/scalers/graphite.md

```diff
@@ -30,6 +30,16 @@ triggers:
 - `threshold` - Value to start scaling for
 - `queryTime` - Query Time to from Seconds/Minutes
 
+### Authentication Parameters
+
+Graphite Scaler supports three types of authentication - basic authentication.
+
+You can use `TriggerAuthentication` CRD to configure the authentication.`authModes: "basic"` Specify `authModes` and other trigger parameters along with secret credentials in `TriggerAuthentication` as mentioned below:
+
+**Basic authentication:**
+- `authMode`: It must contain `basic` in case of Basic Authentication. Specify this in trigger configuration.
+- `username`: This is a required field. Provide the username to be used for basic authentication.
+- `password`: Provide the password to be used for authentication. For convenience, this has been marked optional, because many applications implement basic auth with a username as apikey and password as empty.
```

Comment:
```suggestion
- `password`: Password to be used for authentication. For convenience, this has been marked optional, because many applications implement basic auth with a username as apikey and password as empty.
```